packages/config/src/projects/layer2s/polygonpos.ts

```diff
@@ -43,9 +70,12 @@ export const polygonpos: Bridge = {
         'https://instagram.com/0xpolygon/',
       ],
     },
+    activityDataSource: 'Blockchain RPC',
     description:
-      'Polygon PoS it the official bridge provided by the Polygon team to bridge assets from Ethereum to Polygon chain. The bridge is validated by Polygon validators and allows for asset as well as data movement between Polygon and Ethereum.',
-    category: 'Token Bridge',
+      'Polygon PoS is an EVM-compatible, proof of stake sidechain for Ethereum, planning to become a Validium with a state validating bridge. The bridge is currently validated by Polygon validators and allows for asset as well as data movement between Polygon and Ethereum.',
```

Comment:
proof of stake sidechain OF Ethereum? or is the 'for' intentional?

Reply:
it's from their docs so i'll leave as it is